src/Objs/Homepage/SiteColorsPicker.tsx

```diff
@@ -44,6 +44,30 @@ export const SiteColorsPicker = connect(function SiteColorsPicker({
         </div>
       </div>
 
+      <div className="row">
```

Comment:
* [ ] FYI How about ordering the colors by "priority", i.e., have them by the order/likeliness we expect them to be adjusted? IMO this would be:
1. primary/secondary
2. backdrops
3. text